IntelliNode/test/integration/RemoteSpeechModel.test.js

```diff
@@ -34,6 +38,37 @@ async function testGenerateSpeech() {
   }
 }
 
+async function testOpenAiGenerateSpeech() {
+  try {
+
+    const tempDir = '../temp';
+    if (!fs.existsSync(tempDir)) {
+      fs.mkdirSync(tempDir);
+    }
+
+    const input = new Text2SpeechInput({
+      model: 'tts-1',
+      text: "The quick brown fox jumped over the lazy dog.",
+      voice: "alloy",
+      stream: false
+    });
+
+    const audioContent = await openAiRemoteSpeechModel.generateSpeech(input);
+
+    assert(audioContent.length > 0, 'testGenerateSpeech response length should be greater than 0');
+
+    const decodedAudio = audioHelper.decode(audioContent);
+    const saved = audioHelper.saveAudio(decodedAudio, tempDir, 'temp.mp3');
```

Comment:
the saved file from this test case does not play, kindly verify the reason and fix it.